src/commands/local.ts

```diff
@@ -1018,6 +1018,7 @@ async function communicateWithProcess(
                 processForClasses,
                 localProcess.envVars,
             );
+            throw new UserError("There was an error connecting to the server");
```

Comment:
I would expect that a retry mechanism would not throw error immediately after it tried to recover from error.

Reply:
You are right. After testing it by manually killing the process associated with a class, it did restart the class, but the error was actually lost, it never showed. Since the initial request will fail, and after that we will restart the class, we should at least log an informative message that the request failed but things should work from now. Will include this in my next commit